// react-social-login-buttons/lib/buttons/InstagramLoginButton.js
'use strict';

Object.defineProperty(exports, "__esModule", {
    value: true
});

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _SocialLoginButtonProvider = require('./SocialLoginButtonProvider');

var _SocialLoginButtonProvider2 = _interopRequireDefault(_SocialLoginButtonProvider);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var defaults = {
    text: 'Login with Instagram',
    icon: 'instagram',
    style: {
        background: "linear-gradient(to right, rgb(236, 146, 35) 0%, rgb(177, 42, 160) 50%, rgb(105, 14, 224) 100%)"
    },
    activeStyle: {
        background: "linear-gradient(to right, rgb(214, 146, 60) 0%, rgb(160, 11, 143) 50%, rgb(56, 10, 115) 100%)"
    }
};
/**
 * Instagram login button.
 * For props check {@link SocialLoginButton}
 * @extends {SocialLoginButton}
 */
var InstagramLoginButton = function InstagramLoginButton(props) {
    return _react2.default.createElement(_SocialLoginButtonProvider2.default, { defaults: defaults, props: props });
};

exports.default = InstagramLoginButton;
